backend/lib/cache/index.js

```diff
@@ -102,13 +106,20 @@ module.exports = {
   getControllerRegistrations () {
     return cache.getControllerRegistrations()
   },
+  getResourceQuotas () {
+    return cache.getResourceQuotas()
+  },
   findProjectByNamespace (namespace) {
-    const project = cache.get('projects').find(['spec.namespace', namespace])
+    const project = cache.get('core.gardener.cloud/projects').find(['spec.namespace', namespace])
     if (!project) {
       throw new NotFound(`Namespace '${namespace}' is not related to a gardener project`)
     }
     return project
   },
+  findResourceQuotasByNamespace (namespace) {
```

Comment:
The name of this method it wrong. But however I think we should remove it completely do the following where it is use:
```js
const resourceQuotas = _.filter(cache.getResourceQuotas(), ['metadata.namespace', namespace])
```